strawberry/flask/tests/test_graphqlview.py

```diff
@@ -0,0 +1,56 @@
+import json
```

Comment:
can you move this file under `/tests`? :)

Reply:
Sure